src/sections/Meshery/Meshery-mange-mesh/index.js

```diff
@@ -46,7 +44,7 @@ const MesheryManageMesh = () => {
             </Button>
           </Col>
           <Col sm={12} lg={6}>
-            <StaticImage src={MultiMesh} alt="Meshery multi-mesh" className="multi-mesh-img"/>
+            <img src={MultiMesh} alt="Meshery multi-mesh" className="multi-mesh-img"/>
```

Comment:
Revert this change. Why is this done?

Reply:
If StaticImage is used on .svg it gets blurred. If you will notice the image is blurred out but after changing it to img the image's quality got better.

Reply:
Ok, SVG!
Cool